lib/DDG/Spice/Pokemon/Data.pm

```diff
@@ -21,9 +21,11 @@ spice alt_to => {
 	}
 };
 
+my $accepted_names = join "|", share('pokemon-names.txt')->slurp(chomp => 1);
+
 # Handle statement
 handle remainder => sub {
-    return lc $_ unless !$_ or $_ =~ /\s/;
+    return lc $_ unless !$_ or $_ =~ /\s/ or !($_ =~ /\b($accepted_names)\b/i);
```

Comment:
@sahildua2305 this _**is**_ case insensitive 👍 

`handle remainder_lc` forces the query input to be lowercased. so `Pokemon CHARIZARD` would have `$_ == "charizard"`, and "charizard" exists in our hash :)